packages/help-center/src/components/help-center-header.tsx

```diff
@@ -70,56 +71,87 @@ const HelpCenterHeader: React.FC< Header > = ( {
-						<Button
-							className={ 'help-center-header__maximize' }
-							label={ __( 'Maximize Help Center', __i18n_text_domain__ ) }
-							icon={ chevronUp }
-							tooltipPosition="top left"
-							onClick={ requestMaximize }
-						/>
-					) : (
+		<>
+			<CardHeader className={ classNames }>
+				<Flex>
+					<p id="header-text" className="help-center-header__text">
+						{ isMinimized ? (
+							<Switch>
+								<Route path="/" exact>
+									{ __( 'Help Center', __i18n_text_domain__ ) }
+								</Route>
+								<Route path="/contact-options">
+									{ __( 'Contact Options', __i18n_text_domain__ ) }
+								</Route>
+								<Route path="/contact-form" component={ SupportModeTitle }></Route>
+								<Route path="/post" component={ ArticleTitle }></Route>
+							</Switch>
+						) : (
+							__( 'Help Center', __i18n_text_domain__ )
+						) }
+						{ isMinimized && unreadCount ? (
+							<span className="help-center-header__unread-count">{ formattedUnreadCount }</span>
+						) : null }
+					</p>
+					<div>
+						{ isMinimized ? (
+							<Button
+								className={ 'help-center-header__maximize' }
+								label={ __( 'Maximize Help Center', __i18n_text_domain__ ) }
+								icon={ chevronUp }
+								tooltipPosition="top left"
+								onClick={ requestMaximize }
+							/>
+						) : (
+							<Button
+								className={ 'help-center-header__minimize' }
+								label={ __( 'Minimize Help Center', __i18n_text_domain__ ) }
+								icon={ lineSolid }
+								tooltipPosition="top left"
+								onClick={ onMinimize }
+							/>
+						) }
+
 						<Button
-							className={ 'help-center-header__minimize' }
-							label={ __( 'Minimize Help Center', __i18n_text_domain__ ) }
-							icon={ lineSolid }
+							className={ 'help-center-header__close' }
+							label={ __( 'Close Help Center', __i18n_text_domain__ ) }
 							tooltipPosition="top left"
-							onClick={ onMinimize }
+							icon={ closeSmall }
+							onClick={ onDismiss }
 						/>
-					) }
-
-					<Button
-						className={ 'help-center-header__close' }
-						label={ __( 'Close Help Center', __i18n_text_domain__ ) }
-						tooltipPosition="top left"
-						icon={ closeSmall }
-						onClick={ onDismiss }
-					/>
-				</div>
-			</Flex>
-		</CardHeader>
+					</div>
+				</Flex>
+			</CardHeader>
+			{ ! isMinimized && (
+				<TabPanel
+					className="help-center-header__tab-panel"
+					initialTabName="help"
+					tabs={ [
+						{
+							name: 'nextSteps',
+							title: 'Next Steps',
+							className: 'help-center-container__next-steps-tab',
+						},
+						{
+							name: 'help',
+							title: 'Help',
+							className: 'help-center-container__help-tab',
+						},
+					] }
+					onSelect={ ( tab ) => {
+						if ( tab === 'nextSteps' ) {
+							history.push( '/next-steps-tutorials' );
+						} else if ( tab === 'help' ) {
+							history.push( '/' );
+						}
+					} }
+				>
+					{ () => {
+						return <></>;
```

Comment:
The TabPanel is a super needy 😞

It demands a direct child prop, or else it throws a runtime error, which is why I pass this no-op here. The reason we don't need direct children is because the help center feature already implements a `react-router` MemoryRouter to dynamically render content.

Curious if anyone thinks this is a big deal 🤷

Reply:
> It demands a direct child prop, or else it throws a runtime error, which is why I pass this no-op here. The reason we don't need direct children is because the help center feature already implements a react-router MemoryRouter to dynamically render content.

To elaborate on why we I opted to not use direct children props, I'll describe the alternative approach where we do use them.

```js
// help-center-container.tsx

// existing implementation in this PR
<Card>
  <HelpCenterHeader /> // <--- TabPanel component lives in here
  <HelpCenterContent /> 
  <HelpCenterFooter />
</Card>

// alternative implementation
<Card>
  <HelpCenterHeader />
  <TabPanel>
    { ( tab ) => {
      if ( tab === 'nextSteps' ) {
        return <NextStepsTutorials />
      } else if ( tab === 'help' ) {
        return (
          <>
            <HelpCenterContent /> 
            <HelpCenterFooter />
          </>
        )
      }
    } }
  <TabPanel />
</Card>
```
- HelpCenterContent already has a `Switch` and `Routes` that alternate between all possible views that can be displayed in the help center ( contact form, contact options, inline chat, etc. )
- In this PR, we simply add one more route to `HelpCenterContent` for the Next Steps Tutorial view.
- If we

Reply:
> Im still a little unsure why we are routing at all here (previous to this PR) when it seems like everything could fit into the tab-group/tab-panel/tab-children component paradigm

I might be missing something, but the routing actually makes sense to me. Tabs are traditionally meant to switch between content at identical hierarchical levels. Some of the views in the help center reflect a nested navigation structure, which, to me, lends itself to a solution like `react-router`

Ex. We navigate to the inline chat view by clicking on the "Still need help?" button _and then_ the "Live Chat" button
![2022-06-27 10 22 04](https://user-images.githubusercontent.com/5414230/175999204-bacbbef4-3a44-4d82-af8e-c40be68bbfaf.gif)

@alshakero might be able to speak more to this though

Reply:
I don't think its necessarily any issue I was just curious.

> Tabs are traditionally meant to switch between content at identical hierarchical levels. Some of the views in the help center reflect a nested navigation structure, which, to me, lends itself to a solution like react-router

At my initial look, I would see the tabs and their content containers as entirely separated concerns from one another.  The tab for 'Help' has tab content that has some nested views levels and would require some sort of state/view management (component level, redux, router, etc.). 

Then there is the tab for "Next Steps" which has its own content which may require its own internal levels of navigation. But neither of these tab content containers share any of the same levels and views to they?  We are never going to be on the 'Help' tab and showing content from 'Next Steps' of vice-versa. Their contents are entirely separate and independent. So my initial expectations would be to see the state management solutions for these contents internal to the tab content containers they are concerned with, as opposed to wrapping the entire system with a router and controlling the tabs themselves with it.

Reply:
All that said, thats just my initial thoughts on how I would go about managing the tabs and their content. I don't think the router this way is necessarily a bad approach or a problem, as long as it works and we are confident development on either side wont have adverse effects on the other.

Reply:
Ah I forgot to reply to this yesterday 🤦 Sorry Addie! I'll try to be more thorough moving forward.

> So my initial expectations would be to see the state management solutions for these contents internal to the tab content containers they are concerned with, as opposed to wrapping the entire system with a router and controlling the tabs themselves with it.

I think this is a fair point. Giving it more thought, I don't feel very strongly about either the react-router or the tabs panel local state solution, because, if, as you say, "We are never going to be on the 'Help' tab and needing to show content from 'Next Steps' of vice-versa," then it will be easy to refactor one solution into the other in the future.

I'm fine switching over to the alternative implementation as described in https://github.com/Automattic/wp-calypso/pull/64879#discussion_r906437778 if we'd prefer that

@escapemanuele thoughts?

Reply:
My main thought here is flows that re-enter the Help Center after it's been closed. Will state be maintained? Is that desirable?

My main scenario is like this:

1. The Help Center closes when a Tour is started. 
2. The Tour is completed and Help Center opens to cross off a Step. 
3. Is the Next Steps tab maintained or does it reset to Help? Currently it resets when re-opened.

If routing helps us to maintain that state, we should use it. I'm sure we can maintain that state without it, too. And we should probably not maintain state when we _haven't_ started a Tour after clicking into Next Steps. Tricky.

Reply:
> Ah I forgot to reply to this yesterday 🤦 Sorry Addie! I'll try to be more thorough moving forward.

No worries at all, I see this thread as more of a discussion and not any urgent blocker.

> I'm fine switching over to the alternative implementation as described in https://github.com/Automattic/wp-calypso/pull/64879#discussion_r906437778 if we'd prefer that.

What do you think makes sense? Im fine keeping it the way it is now if we think it makes sense and will continue to work well for our uses. As you noted if we do find need to change it later on

> then it will be easy to refactor one solution into the other in the future.

Reply:
>  is flows that re-enter the Help Center after it's been closed. Will state be maintained? Is that desirable?

Good questions and important to consider! I think both approaches give us ways to handle this, but perhaps the router makes it easier to manage.

We can leave this change to a follow-up issue, and will probably need some clarity from design. I opened a P2 and pinged Nuria and vertex team for thoughts. paYE8P-1Ii-p2

Reply:
> What do you think makes sense?

I'm in camp `react-router`

I agree that local state management makes the most intuitive sense as a developer, but on the flip side, I don't see the usage of `react-router` for these tab panels as unintuitive.

Although not currently designed or on a roadmap, I also wouldn't be surprised if views _do_ eventually become shared within each of the tabs as the help center evolves.

> We are never going to be on the 'Help' tab and needing to show content from 'Next Steps' of vice-versa. Their contents are entirely separate and independent. 

Ex. We might add a "Get Help" button to the next steps tutorial panel that opens the live chat window ( which "lives" in the help tab ). In this case, state managment with `react-router` would make the issue trivial -- create a link component that switches routes. If we were using local state management, we'd need to know to programatically render the help tab first to ensure the inline chat view is available _and then_  switch routes.

As mentioned above, if we end up needing to maintain Help Center state after it's been closed, `react-router` will probably make that easier as well.

Reply:
> Is the Next Steps tab maintained or does it reset to Help? Currently it resets when re-opened.

I added [some thoughts](https://cylonp2.wordpress.com/2022/06/30/help-center-maintaining-state-on-re-open/#comment-1577) on the post but essentially, If we can maintain the last state of Next Steps when reopening the modal, that would be great for the users - so they can pick up on the last thing they did.

> And we should probably not maintain state when we haven't started a Tour after clicking into Next Steps. 

I also agree with that. If the user did not engage at all with Next Steps, perhaps we can just reset it back to Help Center. If that make sense?

Reply:
Lets leave the state preservation to a follow-up PR.

>  If the user did not engage at all with Next Steps, perhaps we can just reset it back to Help Center. If that make sense?

@nuriapenya - It does, but I wonder if its that important? From a technical standpoint, updating it to maintain its state between closing/opening is a rather simple concern - essentially keeping the components 'aware' even when they aren't visible. Something we will also need in the future if we want to provide links from MyHome or LaunchPad to send users into the editor and kickstart a specific tutorial flows.  

However, trying to track user actions and evaluating what they did and did not engage with since the last time they selected a tab to determine what tab we should route them to the next time the center is opened seems like it will be a bit of a complicated system open to some awkward edge cases and weird bugs - for not very much benefit. If we keep it simple and a user clicks "next steps" tab, doesn't do anything within that tab, closes and reopens the help center in the same editing session, and is still on the "next steps" tab... I don't think thats much of an issue. 

Mind you this state preservation im speaking of is within the course of 1 editor session, not after closing and re-opening the editor. When that is the case I think it makes sense to have "Help" as the default landing state, unless we are talking about a specific entry flow that is intended to kick-off a tutorial.

Reply:
>If we keep it simple and a user clicks "next steps" tab, doesn't do anything within that tab, closes and reopens the help center in the same editing session, and is still on the "next steps" tab... I don't think thats much of an issue.

True. I thought resetting back to the Help centre was the simple approach.

> Mind you this state preservation im speaking of is within the course of 1 editor session, not after closing and re-opening the editor. 

Gotcha 👌🏽 That makes total sense, it should be always while users are interacting with the Editor.